_gsocproposals/2021/proposal_TMVADLConv3DGPU.md

```diff
@@ -0,0 +1,38 @@
+---
+title: TMVA Deep Learning Developments - 3D Convolutions for GPU
+layout: gsoc_proposal
+project: TMVA
+year: 2020
+organization:
+  - CERN
+  - OProject
+---
+
+# Description
+
+Toolkit for Multivariate Analysis (TMVA) is a multi-purpose machine learning toolkit integrated into the ROOT scientific software framework, used in many particle physics data analysis and
+applications. We have expanded TMVA's capabilities to include a deep learning library (DNN) supporting 2Dconvolutional layers with training on both CPU and GPUs. This summer we would like to complete it
+with the 3D convolution operator.
+3D CNNs have very promising applications in particle physics, such as imaging calorimetry and particle tracking, allowing physicists to use new techniques to identify particles and search for new physics. In addition they can be used as Generative Adversarial Networks for fast imaging detector simulations.
+
+
+
+
+## Task ideas and expected results
+The initial task of the student will be validating and integrating in ROOT the  CPU implementation for 3D convolution that was developed last summer .
+Afterwards, the main task will be the development of the 3D convolution for Nvidia GPU using the cuDNN library, following the current implementation present in ROOT/TMVA for the 2D operator. 
+The developed code should support both the training and inference for deep learning models.
+
+## Requirements 
+Strong C++ skills, solid knowledge of deep learning, familiarity with GPUs
+
+## Mentors
```

Comment:
Lorenzo, you need to :  
- add Omar in `gsoc/2021/mentors.md` file
- move your name to the right place (entries must be sorted in **last name** alphabetic order)

Reply:
I have update the mentor list, 
Cheers
L.

Reply:
Thanks! But do you want to add Sitong as a mentor in the proposal?

Reply:
No, Sitong will not be mentoring this student.